src/__tests__/components/Navigation.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import store from '../../__mocks__/configureStore';
+import Navigation from '../../components/Navigation';
+
+describe('CNavigation snapshot', () => {
+  test('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
+          <BrowserRouter>
+            <Navigation />
```

Comment:
Please add a `Back` button on the `Details` page which allows the user to go to the `Home` page.

Reply:
Ok, done.